mittab/apps/tab/views.py

```diff
@@ -248,4 +272,47 @@ def view_scratches(request):
                              {'item_type':'team',
                               'title': "Viewing All Scratches for Teams",
                               'item_list':c_scratches}, context_instance=RequestContext(request))
+
+def upload_data(request):
+    if request.method == 'POST':
+      form = UploadDataForm(request.POST, request.FILES)
+      if form.is_valid():
+        judgeErrors = scratchErrors = teamErrors = []
```

Comment:
nix the camelCase (sorry that some of the other files have it, a former contributor wrote a lot of java)